conda/cli/install.py

```diff
@@ -251,6 +251,9 @@ def install(args, parser, command="install"):
     if num_cp:
         if num_cp == len(args_packages):
             explicit(args_packages, prefix, verbose=not context.quiet)
+            if newenv:
+                touch_nonadmin(prefix)
```

Comment:
Why is the `touch_nonadmin` needed?

Reply:
`menuinst` shenanigans. `$PREFIX/.nonadmin` is used as a marker to avoid trying a superuser elevation on Windows. It's... messy, but unfortunately legacy behavior that we need to maintain for backwards compatibility. I think @marcoesters looked into it not long ago, and we can probably remove this at some point.

That said, this change here is only making things consistent with the other logic in this module that end up creating new environments.

Reply:
Yes, `menuinst` has the (in my opinion unfortunate) design choice that it's admin by default unless it finds that `.nonadmin` file. Writing it into a new environment makes sense to me, or you could install `menuinst` into the new environment and risk privilege escalation.

> we can probably remove this at some point.

That's a separate conversation, but this will be very hard to do without breaking installations with older menuinst versions if we do a 180 here. That's a major breaking change.